### ISSUE TITLE
CVDD Finetuning

### ISSUE THREAD

**Issue by seungyonglee0802** [BoyuanJiang/FitDiT#7]
Thanks again for your nice work.

I have a question on the part that finetuning 1024-pretrained model with 1536 resolution images.
> For quantitative comparison
on CVDD, we first train a model with the combination of
VITON-HD and DressCode at a resolution of 1024 × 768
and then fine-tune it at a higher resolution of 1536 × 1152
with only the parameters of the attention layers being train-
able

Did you also **finetune the GarmentDiT** with 1536 resolution images in CVDD dataset? or **finetune the DenoisingDiT only**?
Also, if you only finenune the DenoisingDiT, did you use 1024 resolution garment images for the pretrained GarmentDiT and 1536 resolution target images to train DenoisingDiT?

Thanks in advance 👍

**Comment by BoyuanJiang** [comment]
For quantitative comparison on CVDD we only fine-tune the DenoisingDiT at a resolution of 1536x1152, using a combination of VITON-HD and DressCode by upsampling them to this resolution. The GarmentDiT is only pretrained at 1024x768 during the first stage; however, we may achieve better results if we pretrain the GarmentDiT at 1536x1152.

**Comment by Holmes2002** [comment]
@BoyuanJiang Hi, I want to ask a question that how much factor weight you used between noise loss and frequency loss. I recoded as u implemented in paper. However, frequence FFT loss seem too high. Can u help me ?
